### PR TITLE
Fix Coverity issues

### PR DESCRIPTION
**Pull request by temanbrcom**

### CODE REVIEW

**Review by grianbrcom — APPROVED** [review]
:see_no_evil:

**Review by asatklichov — APPROVED** [review]
📆

File: server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/statements/SetToBooleanStatement.java [hunk 1]
```diff
@@ -29,6 +30,7 @@
 import static java.util.stream.Collectors.toList;
 
 /** This class implements the logic for SET TO TRUE/FALSE statement. */
+@EqualsAndHashCode(callSuper = false)
```
Comment (grianbrcom) [inline server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/statements/SetToBooleanStatement.java:33]: Why? This means that equals will work only with `receivingFields`.

**Review by grianbrcom — CHANGES_REQUESTED** [review]
It doesn't make sense.

**Review by grianbrcom — APPROVED** [review]
:confetti_ball:

**Pull request merged**